File: script_sadaf.R
##Write a script that prints your name, your email, your slack username (with @) and your biostack, twitter_handle, hamming_distance in that order.##



name <- "Sadaf Raza"
email <- "sadafraza48@gmail.com"
biostack <- "Genomics"
slack_username <- "@sadaf"
twitter_handle <- "@svdvf"




data <- data.frame(Information=c(name, email, slack_username, biostack, twitter_handle))

cat(paste(data, collapse = ','))

File: script_sadaf.py
#!/usr/bin/env python3
# coding: utf-8

# In[1]:


print("NAME: Sadaf Raza \nE-MAIL: sadafraza48@gmail.com \nSLACK USERNAME: @sadaf \nBIOSTACK: Genomics \nTwitter Handle: @svdvf")
def hamming_distance(a,b):
    count=0
    for i in range(len(a)):
        if a[i] != b[i]:
            count +=1
    return count

print(hamming_distance('@sadaf','@svdvf'))


# In[ ]:




